core/commands/store/upload/init.go

```diff
@@ -0,0 +1,248 @@
+package upload
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	cmds "github.com/TRON-US/go-btfs-cmds"
+	config "github.com/TRON-US/go-btfs-config"
+	"github.com/TRON-US/go-btfs/core/commands/cmdenv"
+	"github.com/TRON-US/go-btfs/core/commands/storage"
+	"github.com/TRON-US/go-btfs/core/corehttp/remote"
+	"github.com/TRON-US/go-btfs/core/escrow"
+	"github.com/TRON-US/go-btfs/core/guard"
+	"github.com/cenkalti/backoff/v3"
+	"github.com/dustin/go-humanize"
+	cidlib "github.com/ipfs/go-cid"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/prometheus/common/log"
+	"github.com/tron-us/go-btfs-common/crypto"
+	escrowpb "github.com/tron-us/go-btfs-common/protos/escrow"
+	guardPb "github.com/tron-us/go-btfs-common/protos/guard"
+	"github.com/tron-us/go-btfs-common/utils/grpc"
+	"strconv"
+	"time"
+)
+
+var bo = func() *backoff.ExponentialBackOff {
```

Comment:
rename this to which part, e.g. `initBo` or something. We should have different `bo` objects for different steps (maybe gets controlled by fsm as well?)